app/api/margatsni/v1/endpoints/users.rb

```diff
@@ -13,6 +13,10 @@ def represent_user_with_token(user)
           def represent_current_user(current_user)
             present :user, current_user, with: Margatsni::V1::Entities::User
           end
+
+          def represent_specific_user(user)
+            present :user, user, with: Margatsni::V1::Entities::User
```

Comment:
Replace 2 methods `represent_current_user` and `represent_specific_user ` with a single one: 
```
  def represent_user(user)
    present :user, user, with: Margatsni::V1::Entities::User
  end
```
No needs to duplicate these methods

Reply:
Done. Thanks!